types/storybook__addon-actions/index.d.ts

```diff
@@ -1,11 +1,12 @@
-// Type definitions for @storybook/addon-actions 3.0
+// Type definitions for @storybook/addon-actions 3.4
 // Project: https://github.com/storybooks/storybook
-// Definitions by: Joscha Feth <https://github.com/joscha>
+// Definitions by: Joscha Feth <https://github.com/joscha>, June <https://github.com/jicjjang>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.6
+// TypeScript Version: 2.9
 
 export type HandlerFunction = (...args: any[]) => undefined;
 export type DecoratorFunction = (args: any[]) => any[];
 
-export function decorateAction(decorators: DecoratorFunction[]): (name: string) => HandlerFunction;
+export function decorateAction(decorators: DecoratorFunction[]): (name: string, options?: object) => HandlerFunction;
```

Comment:
can we make the options more narrow than `object`? There are only three that are allowed: https://github.com/storybooks/storybook/tree/master/addons/actions#available-options

Reply:
I got it :D